apps/tup-cms/Dockerfile

```
@@ -7,7 +7,7 @@ COPY . /code/
 RUN npx nx build tup-ui
 RUN npx nx build tup-cms-react
 
-FROM taccwma/core-cms:v4.17.1
+FROM taccwma/core-cms:e9e49586
```

The fix is released now.

```suggestion
FROM taccwma/core-cms:v4.24.1
```